### PR TITLE
Added sigin sigup pages

### PR DESCRIPTION
**Pull request by papali1305**
tried some effect in header and footer and we should add more pages also 

please accept the PR

### CODE REVIEW

File: backend/app/__pycache__/__init__.cpython-312.pyc
Comment (VincenzoMeschi) [inline backend/app/__pycache__/__init__.cpython-312.pyc:1]: add __pycache__ to `.gitignore` inside of `/backend`

File: frontend/package.json
Comment (VincenzoMeschi) [inline frontend/package.json:1]: What is your reason for updating the version of next?

**Comment by papali1305** [comment]
its better and we should go with the lastest updates

**Comment by papali1305** [comment]
Added about , contact us and add ed some effects in pages and fixed the user linlk

**Pull request closed without merging**